paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/SortCompactAction.java

```diff
@@ -66,6 +66,13 @@ public SortCompactAction(
     public void run() throws Exception {
         StreamExecutionEnvironment env = StreamExecutionEnvironment.getExecutionEnvironment();
         env.setRuntimeMode(RuntimeExecutionMode.BATCH);
```

Comment:
remove this, batch or stream can be set by users using Flink CLI.

Reply:
batch mode may need to forcely specified